src/tests/api/CallbackApi.test.ts

```diff
@@ -18,32 +28,44 @@ describe("Callback API", () => {
 		f2fStubPayload.yotiMockID = yotiMockId;
 		const sessionResponse = await startStubServiceAndReturnSessionId(f2fStubPayload);
 		const sessionId = sessionResponse.data.session_id;
+		const sub = sessionResponse.data.sub;
+
 		// Document Selection
 		const response = await postDocumentSelection(docSelectionData, sessionId);
 		expect(response.status).toBe(200);
+
 		// Authorization
 		const authResponse = await authorizationGet(sessionId);
 		expect(authResponse.status).toBe(200);
+
 		// // Post Token
 		const tokenResponse = await tokenPost(authResponse.data.authorizationCode.value, authResponse.data.redirect_uri );
 		expect(tokenResponse.status).toBe(200);
+
 		// Post User Info
 		const userInfoResponse = await userInfoPost("Bearer " + tokenResponse.data.access_token);
 		expect(userInfoResponse.status).toBe(202);
 
-		// Test Suspended - additional engineering work is required to facilitate the validation of BE systems, designs and US to follow
 		// Get Yoti Session Id
-		// const session = await getSessionById(sessionId, "session-f2f-cri-ddb");
-		// const yotiSessionId: any = session?.yotiSessionId;
-		// console.log(yotiSessionId);
-		// // Yoti Callback
-		// const callbackResponse = await callbackPost(yotiSessionId);
-		// expect(userInfoResponse.status).toBe(202);
-		// Verifiable Credential Validation
-		//await setTimeout(10000);
-		//const jwtToken = await receiveJwtTokenFromSqsMessage();
-		//validateJwtToken(jwtToken, vcResponseData, yotiMockId);
-	});
+		const session = await getSessionById(sessionId, "session-f2f-cri-ddb");
```

Comment:
This will fail in the pipeline as it uses the client to get session from dynamo db.